Jasmine/Grid/ViewControllers/GridGameViewController.swift

```diff
@@ -210,7 +218,7 @@ extension GridGameViewController: BaseGameViewControllerDelegate {
 
     // MARK: Game Status
     /// Notifies the view controller that the game state has changed.
-    func notifyGameStatus(with newStatus: GameStatus) {
+    func notifyGameStatus() {
```

Comment:
if newStatus is removed, this method name doesn't make sense. 
--> notifyGameStatusUpdated()